daemon/main_test.go

```diff
@@ -45,6 +46,36 @@ func (v *DummyPlugin) DeleteBridgePort(deleteRequest *opi.DeleteBridgePortReques
 	return nil
 }
 
+type SriovManagerMock struct {}
```

Comment:
Just a general question whether we will be using Mocks for HW functionality?

Reply:
We should use a stub (or ideally a fake) when dealing with deeper layers of the architecture. We shouldn't blindly mock everything that goes to the hardware. Rather, we should write code that encapsulates behavior.